install.sh

```diff
@@ -19,10 +19,6 @@ echo "Open gridd: $open_gridd"
 
 set -euo pipefail
 
-install_linux_headers() {
-  apt install -y linux-headers-$(uname -r) --no-install-recommends
-}
-
 use_package_manager_with_retries wait_for_apt_locks install_linux_headers 10 3
```

Comment:
This line is calling install_linux_headers, right?
Btw, are we able to test this script somewhere?

Reply:
Yeah I put my testing steps in the PR description. The docker.io/alburgess/aks-gpu:latest has the changes. 

The line installs linux-headers but linux-headers is already installed on the vhd by the time this step occurs so its redundant. And since its an apt install call, it won't be compliant with our efforts to have no egress calls to anything other than mcr.microsoft.com during vhd provisioning

Reply:
Sorry I am not familiar with this test environment. So is the testing steps execute this install.sh?
I was just wondering if the function install_linux_headers() is removed but when line 22 calls it, will it return error like function not defined.
If the test executes install.sh, then I have no other concern.

Reply:
You know the call didn't fail - but I will remove the install_linux_headers function call and re-test to verify for good measure.

And yes the testing measures should execute install.sh :)

Reply:
Removed the function call and verified that it still works as intended :)